apps/mobile/src/utils/types.ts

```diff
@@ -0,0 +1,5 @@
+import { ReactNode } from 'react';
```

Comment:
Should we add a `types` package to share this with the extension? It's not a big deal for one type

Reply:
Was thinking this, but doesn't really belong in models pkg as it's React specific. If anything, maybe a helper type from ui pkg makes sense?